presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -2,40 +2,22 @@ datasource: presto
 query-names: presto/tpch/${query}.sql
 runs: 6
 prewarm-runs: 2
-before-execution: sleep-4s, presto/session_set_reorder_joins.sql
+before-execution: sleep-4s
 frequency: 7
 database: hive
-tpch_small: tpch_10gb_orc
-tpch_medium: tpch_100gb_orc
-tpch_large: tpch_1tb_orc
+tpch_300: tpch_sf300_orc
+tpch_1000: tpch_sf1000_orc
+tpch_3000: tpch_sf3000_orc
 prefix: ""
 variables:
   1:
-    # queries too slow to run on 100gb without reordering
-    query: q2, q8, q9
-    schema: ${tpch_small}
-    reorder_joins: true, false
+    query: q5, q7, q8, q9, q17, q18, q21
+    schema: ${tpch_300}
+
   2:
-    # queries too slow to run on 100gb without reordering
-    query: q8, q9
-    schema: ${tpch_medium}
-    reorder_joins: true
+    query: q10, q12, q13, q15, q1, q20, q2, q3, q4
+    schema: ${tpch_1000}
+
   3:
-    # queries too slow to run on 100gb without reordering
-    query: q2
-    schema: ${tpch_large}
-    reorder_joins: true
-  4:
-    # queries too slow to run on 1tb
-    query: q3, q4, q5, q7, q17, q18, q21
-    schema: ${tpch_medium}
-    reorder_joins: true, false
-  5:
-    query: q10, q11, q12, q13, q14, q15, q16, q19, q20, q22
-    schema: ${tpch_large}
-    reorder_joins: true, false
-  6:
-    # queries without joins
-    query: q1, q6
-    schema: ${tpch_large}
-    reorder_joins: false
+    query: q6, q11, q14, q16, q19, q22
```

Comment:
it would be nice to refactor files like q6 to q06

Reply:
done